WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -39,6 +38,16 @@ public class KeyManager
 
 	private static readonly KeyPath TestNetAccountKeyPath = new("m/84h/1h/0h");
 
+	private static readonly JsonConverter[] JsonConverters =
```

Comment:
It seems to me having the attribute of 

`[JsonConverter(typeof(BitcoinEncryptedSecretNoECJsonConverter))]`

above the property will give the same result. What is the difference?

Reply:
I explain this in a long comment below.